part13.md

```diff
@@ -0,0 +1,26 @@
+## Part 13: Finishing Up (Github)
```

Comment:
Personally I still disagree with including git / GitHub in the tutorials, just because it's a little tangential